docs/section21/Rule21-18.md

```diff
@@ -0,0 +1,55 @@
+
+# Boiler - Rule 21-18  
+
+**Rule ID:** 21-18  
+**Rule Description:** Baseline shall have only one heating hot water plant.  
+**Rule Assertion:** B-RMR LEN(FluidLoop.type == "HEATING") == 1  
+**Appendix G Section:** Section 21 Boiler  
+**Appendix G Section Reference:** Section G3.1.3.2 Building System-Specific Modeling Requirements for the Baseline model  
+
+**Applicability:** All required data elements exist for B-RMR  
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-1, 5, 7, 11(for climate zones other than 0 through 3A), 12, 1a, 7a, 11a(for climate zones other than 0 through 3A), 12a.
+
```

Comment:
based on the revised logic which includes PHW plans, we should probably add systems 1B, 3B, 5B, 6B, 7B, 8B, 9B, 11B, 12B, 13B, 1C, 3C, 7C, 11C, 12C, 13C to the list. @mkarpman please confirm.

Reply:
Yes, I agree.

Reply:
need to add PHW systems (c series)